### PR TITLE
Add namespace to K8sBlockExecutor config

### PR DESCRIPTION
**Pull request by mattcamp**
Add a config param to control block executor namespace

This has been tested locally as much as possible but really needs to be deployed into a kubernetes cluster to work.

### CODE REVIEW

File: mage_ai/services/k8s/config.py [hunk 1]
```diff
@@ -22,6 +23,7 @@ class K8sExecutorConfig(BaseConfig):
     resource_limits: Dict = None
     resource_requests: Dict = None
     service_account_name: str = DEFAULT_SERVICE_ACCOUNT_NAME
+    namespace: str = DEFAULT_NAMESPACE
```
Comment (dy46) [inline mage_ai/services/k8s/config.py:26]: let's add this in alphabetical order

Comment (jamesking-github) [reply mage_ai/services/k8s/config.py]: Updated as requested @dy46

**Comment by jamesking-github** [comment]
Should be good to go now @dy46

**Comment by wangxiaoyou1993** [comment]
Could you also update this doc? https://github.com/mage-ai/mage-ai/blob/master/docs/production/configuring-production-settings/compute-resource.mdx#L43

**Comment by jamesking-github** [comment]
> Could you also update this doc? https://github.com/mage-ai/mage-ai/blob/master/docs/production/configuring-production-settings/compute-resource.mdx#L43

It doesn't look like I have permissions, it would require a fork to propose changes

**Comment by wangxiaoyou1993** [comment]
@jamesking-github You can update the doc in your current git branch.

**Comment by jamesking-github** [comment]
> @jamesking-github You can update the doc in your current git branch.

Realised my mistake, I have updated the docs.  Please confirm all is as you expect.  Thanks

**Review by jamesking-github — APPROVED** [review]

**Pull request merged**